### PR TITLE
use dev-nginx to setup nginx proxy

### PR DESCRIPTION
**Pull request by akash1810**
This is the start of scripting the setup process for amiable. Part I is using [dev-nginx](https://github.com/guardian/dev-nginx) to configure the nginx proxy.

The benefit of this is:
- the removal of certs from the repo
- keeping things DRY (`dev-nginx` does the hard work, so you don't have to)

### CODE REVIEW

**Comment by akash1810** [comment]
cc @guardian/deploy-infrastructure

File: README.md [hunk 1]
```diff
@@ -39,4 +30,4 @@ For this reason, in order to set up locally, the following steps must be followe
  
 ### Common problems
  - If when running master you can an error "Could not find a suitable constructor..." it's something wrong with your
- config file - you probably need to add `include "application.conf"` to you application.local.conf file.
\ No newline at end of file
+ config file - you probably need to add `include "application.conf"` to you application.local.conf file.
```
Comment (katebee) [inline README.md:33]: not sure what changed on this line? But just noticed a missing character:
`to you` should be `to your` 🕵️

File: README.md [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
-=======
+# Amiable
 Amiable is a web app for monitoring the use of AMIs.
 You can access it here:
 https://amiable.gutools.co.uk
 
 
-### How to run Amiable locally
+## How to run Amiable locally
+Amiable uses Google Auth. For this reason, we need to run amiable through an nginx proxy `amiable.local.dev-gutools.co.uk`.
```
Comment (katebee) [inline README.md:8]: Minor: `amiable` should start with a capital letter here

File: README.md (hunk 1 above)
Comment (akash1810) [reply README.md:33]: Think this was the IDE adding whitespace at end of file.

File: README.md (hunk 2 above)
Comment (rtyley) [reply README.md]: Ha @katebee, just wanted to point out that there's quite a nice GitHub ['Suggestion' feature](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) for suggesting changes like these!

image

Comment (katebee) [reply README.md]: thank you, I was not aware! 🥇

**Pull request merged**